docs/index.md

```diff
@@ -0,0 +1,151 @@
+---
+hide_title: true
+sidebar_position: 0
+title: Overview
+---
+
+# Documentation
+
+Macrometa's Global Data Network (GDN) is a distributed backend for building global applications. 
+
+### Getting Started
+
+<grid cols={3}>
+  <card
+    heading="Quickstart Guide"
+    description="Get started with a comprehensive tutorial that covers the basics."
+    href="/quickstart"
+  />
+  <card
+    heading="Developer Tools"
+    description="Leverage our CLI, SDKs and libraries to start building now."
+    href="/development"
+  />
+  <card
+    heading="Sample Apps"
+    description="Explore our library of open source sample apps and starter projects."
+    href="/apps"
+  />
+</grid>
+
+### Global Data Mesh
+
+Store and serve any kind of data, anywhere in the world, with high throughput and ultra-low latency reads and writes.
+
+<grid cols={3}>
+  <card
+    heading="Graphs"
+    description="Dynamic, high-throughput graph store with ultra low-latency queries."
+    href="/graphs"
+  />
+  <card
+    heading="Search"
+    description="Flexible full-text search and similarity ranking."
+    href="/search/search"
+  />
+  <card
+    heading="Streams"
+    description="Distributed, real-time stream engine with pub/sub and message queues."
+    href="/streams"
+  />
+</grid>
+
+#### Collections
+
+<grid cols={3}>
+  <card
+    heading="Intro to Collections"
+    description="Flexible, poly-model data layer supporting multiple data types."
+    href="/collections"
+  />
+  <card
+    heading="Document Collections"
+    description="NoSQL document collections with flexible indexing and modeling."
+    href="/collections/documents"
+  />
+  <card
+    heading="Dynamo-mode Collections"
+    description="Full-featured compatibility with AWS DynamoDB SDKs and CLI."
+    href="/collections/dynamo/create-dynamo-table"
+  />
+  <card
+    heading="Graph Edge Collection"
+    description="High-throughput KV collections powering a wide range of use cases."
+    href="/collections/keyvalue"
+  />
+  <card
+    heading="Key-Value Collections"
+    description="High-throughput KV collections powering a wide range of use cases."
+    href="/collections/keyvalue"
+  />
+  <card
+    heading="Redis-mode Collections"
+    description="Full-featured Redis-compatible collection and API."
+    href="/collections/redis-mode/"
+  />
+</grid>
+
+### Edge Compute
+
+Deploy compute runtimes and complex event-driven workloads alongside your data, and within milliseconds of your customers.
+
+<grid cols={3}>
+  <card
```

Comment:
Queries and query workers are not really separate things. Maybe like to SQL or C8QL here to fill out the cards?

Reply:
@diana-macrometa I'll remove the first tile for now, but we should revisit this. Queries and Query Workers are definitely distinct things with distinct use cases, though we've structured them as one thing here and in the GUI.

Reply:
Depends on which "query" you are talking about, because we have:

- queries you run in the Query Workers tab (or CLI or API or whatever)
- query workers that you save
- ad hoc queries for stream workers
- queries that are part of stream workers
- table queries (stream workers)
- window queries (stream workers)

The documentation for all of which I have been hammering into submission for the past few weeks.

Reply:
@diana-macrometa totally get that, truly some heroic work there!